proto/gk/v1/access.proto

```diff
@@ -42,6 +43,12 @@ service Access {
 			body : "*"
 		};
 	}
+
+	rpc RemovePolicyIdentity(AccessRemovePolicyIdentityRequest) returns (google.protobuf.Empty) {
```

Comment:
Better to avoid using `google.protobuf.Empty` to keep API compatibility and create a specific response type (similar to `AccessAddPolicyIdentityResponse`).